openstack/cinder/templates/type-seeds.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Release.IsUpgrade }}
+{{- if .Capabilities.APIVersions.Has "openstack.stable.sap.cc/v1"}}
```

Comment:
This is fine, I guess. The `cinder-seed` does not have it though, so cinder can currently not be deployed without that `CustomResourceDefinition` existing.

You should add a space before the closing `}}`, though.

Reply:
updated